src/uu/cksum/src/cksum.rs

```diff
@@ -181,11 +207,16 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         matches.get_flag(options::BASE64),
     );
 
+    // Print hardware debug info if requested
+    if matches.get_flag(options::DEBUG) {
+        print_cpu_debug_info();
+    }
+
     let opts = ChecksumComputeOptions {
         algo_kind: algo,
         output_format,
         line_ending,
-        binary: false,
```

Comment:
This is not related to --debug, and changes a behavior that is not yet settled (see #9168).

Please revert this, we can discuss it somewhere else